js/script.js

```diff
@@ -4,11 +4,19 @@ import {
   $taskList,
   $btnCreate,
   $searchInput,
+  $filterAll,
+  $filterComplete,
+  $filterIncomplete,
+  $btnLoadMore,
+  $btnShowLess,
 } from "../js/elements.js";
 import { stripSanitizedParts } from "../js/utils/stripSanitizedParts.js";
 import { formatDate } from "../js/helpers/formatDate.js";
 
 let tasks = [];
+let page_current = 1;
+const page_load = 9;
+let currentFilter = "all";
```

Comment:
```suggestion
let currentPage = 1;
let currentFilter = ALL;

const TASK_PER_PAGE = 9;
```
 
- Make these `"all"`, `"incomplete"`, `"complete"` string constant and move those to `constants.js` file.
- Also move   `TASK_PER_PAGE` to the same `constants.js` file.